src/main/scala/com/typesafe/sbt/duplicates/DuplicatesFinderPlugin.scala

```diff
@@ -20,31 +22,73 @@ object DuplicatesFinderPlugin extends AutoPlugin {
     excludePatterns := List("^META-INF/.*", "reference.conf"),
     reportDuplicatesWithSameContent := false,
     includeBootClasspath := false,
+    reportDuplicates := reportDuplicates0.value,
     checkDuplicates := checkDuplicates0.value
   )
+  // Borrowed from https://github.com/jrudolph/sbt-dependency-graph/master/src/main/scala/net/virtualvoid/sbt/graph/DependencyGraphSettings.scala
+  // This is to support 0.13.8's InlineConfigurationWithExcludes while not forcing 0.13.8
+  type HasModule = {
+    val module: ModuleID
+  }
 
-  private lazy val checkDuplicates0 = Def.task {
-    val log = streams.value.log
-    val reportSameContent = reportDuplicatesWithSameContent.value
-    val additionalClasspath = if(includeBootClasspath.value) bootClasspath else Seq.empty
+  def crossName(ivyModule: IvySbt#Module) =
+    ivyModule.moduleSettings match {
+      case ic: InlineConfiguration ⇒ ic.module.name
+      case hm: HasModule if hm.getClass.getName == "sbt.InlineConfigurationWithExcludes" ⇒ hm.module.name
+      case _ ⇒
+        throw new IllegalStateException("sbt-dependency-graph plugin currently only supports InlineConfiguration of ivy settings (the default in sbt)")
```

Comment:
please amend the exception message to not reference sbt-dependency-graph ;)

Reply:
Of course. Cut-and-paste mistake.